.changelog/1632.fixed.txt

```diff
@@ -0,0 +1 @@
+fix(opamp): Add the Default provider to the provider list in ConfigProviderSettings
```

Comment:
It's envprovider

Reply:
according to the otel spec the env provider is the default

Reply:
Hmm, isn't fileprovider default as well?